app/src/routes/routesMap.js

```diff
@@ -348,6 +352,8 @@ export const onBeforeRouteChange = (dispatch, getState, { action }) => {
     !isAdminNewPageType
   ) {
     if (isProjectExists) {
+      const organization = organizations.find(({ slug }) => slug === hashOrganizationSlug);
+      dispatch(setActiveOrganizationAction(organization));
```

Comment:
Please add a TODO comment, that it will be replaced with request for organization by slug.
As for the navigating between pages we will not always have an array of all organizations on the instance.

Reply:
if there is no organization, then active organization will be undefined.
Done.